frontend/src/app/modules/availability/slot/slot.component.ts

```diff
@@ -87,9 +93,49 @@ export class SlotComponent extends BaseComponent implements OnDestroy {
         this.isDeleted.emit(isRemove);
     }
 
+    public changeSlotActivityClick(event: MatSlideToggleChange) {
+        this.confirmWindowService.openConfirmDialog({
+            buttonsOptions: [
+                {
+                    class: 'confirm-accept-button',
+                    label: 'Yes',
+                    onClickEvent: this.changeActivityEventEmitter,
+                },
+                {
+                    class: 'confirm-cancel-button',
+                    label: 'Cancel',
+                    onClickEvent: this.cancelActivityEventEmitter,
+                },
+            ],
+            title: event.checked ? 'Confirm Slot Activation' : 'Confirm Slot Inactivation',
+            message: event.checked ? activationSlotMessage : inactivationSlotMessage,
+        });
+    }
+
+    public changeSlotActivity() {
+        this.http
+            .updateSlotEnabling(this.slot?.id)
+            .pipe(this.untilThis)
+            .subscribe(
+                () => {
+                    this.notifications.showSuccessMessage('Slot`s activity was successfully changed');
+                    window.location.reload();
```

Comment:
Why do we need this?

Reply:
Notification or reloading?

Reply:
![image](https://user-images.githubusercontent.com/71783287/186978666-5892bdaa-f11c-4bbd-8fcc-da1bfba0221a.png)

Reply:
reloading